port: /dev/ttyDXL

joint_order: [Joint1, Joint2, Joint3, Joint4, Joint5, Joint6, gripper]
sleep_positions: [0, 0, 0, 0, 0, 0, 0]

joint_state_publisher:
  update_rate: 30
  publish_states: true
  topic_name: joint_states

groups:
  arm: [Joint1, Joint2, Joint3, Joint4, Joint5, Joint6]

grippers:
  gripper:
    horn_radius: 0.022
    arm_length: 0.036
    left_finger: GL
    right_finger: GR

shadows:
  Joint1:
    shadow_list: [Joint1_shadow]
    calibrate: true

sisters:

motors:
  Joint1:
    ID: 1
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 0
    Velocity_Limit: 131
    Min_Position_Limit: 0
    Max_Position_Limit: 4095
    Secondary_ID: 255
    
  Joint1_shadow:
    ID: 2
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 0
    Velocity_Limit: 131
    Min_Position_Limit: 0
    Max_Position_Limit: 4095
    Secondary_ID: 1

  Joint2:
    ID: 3
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 1
    Velocity_Limit: 131
    Min_Position_Limit: 841
    Max_Position_Limit: 2867
    Secondary_ID: 255

  Joint3:
    ID: 4
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 1
    Velocity_Limit: 131
    Min_Position_Limit: 898
    Max_Position_Limit: 3094
    Secondary_ID: 255

  Joint4:
    ID: 5
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 1
    Velocity_Limit: 131
    Min_Position_Limit: 830
    Max_Position_Limit: 3504
    Secondary_ID: 255

  Joint5:
    ID: 6
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 0
    Velocity_Limit: 131
    Min_Position_Limit: 0
    Max_Position_Limit: 4095
    Secondary_ID: 255
    
  Joint6:
    ID: 7
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 0
    Velocity_Limit: 131
    Min_Position_Limit: 0
    Max_Position_Limit: 4095
    Secondary_ID: 255

  gripper:
    ID: 8
    Baud_Rate: 3
    Return_Delay_Time: 0
    Drive_Mode: 0
    Velocity_Limit: 131
    Min_Position_Limit: 0
    Max_Position_Limit: 4095
    Secondary_ID: 255
